docs/Creating-Components.md

```diff
@@ -0,0 +1,150 @@
+```
+
+The schema check currently only happens on local data. Eventually, we hope to run the schema validator at runtime.
+
+### Update some local fixture data
+Pick a JSON file under `data/test/news/articles/[id].json`, and:
+
+1) add an example of your block somewhere in the `content.model.blocks` array.
+2) add your new component to the `blockTypes` array.
+
+Run `npm run dev` and you should see your component at your article of choice, eg http://localhost:7080/news/articles/c0000000001o
+
+### Create the container
+We've added our _component_, which should be kept as simple as possible. Now we need to create our _container_, which contains the business logic for mapping Optimo block data to the React parameters our component needs.
+
+Add a new folder under `src/app/containers/[Component Name]/`. You will need:
+
+* index.jsx - describes the mapping of Optimo block data to React parameters
+* index.test.jsx - creates "snapshots" of the component with the various different rendered outputs for the business logic in the container
+
+This step is quite complicated, so copy and paste from a similar example and tweak the code to your requirements.
+
+As part of this work, you'll need to...
+
+### Define your component propTypes
+Under `src/app/models/propTypes/[Component Name (lowercase)]`, create an `index.jsx` which describes the propTypes your component needs. Import this into your `src/app/containers/[Component Name]/index.jsx`.
+
+## Part 3: Integrate the component into the Article
+Before your component can be used in production, it must go through a full accessibility and UX review. These steps don't happen as much in the first two PRs as we don't want them to become a bottleneck for development.
+
+### Build dependency into article renderer
+As part of the final review process, you'll need to import your component's container and add it to the `componentsToRenderMain` object in `src/app/containers/Article/index.jsx`.
+
+This feels like duplication of the schema - and it kind of is - but is currently our solution for preventing bundling _all components_ into the application JS.
+
+### Testing
+Before merge, if your new component requires integration testing it should be covered in the end-to-end (E2E) tests.
+
+The test team is responsible for writing integration tests - [which live in Simorgh](https://github.com/BBC-News/simorgh/tree/latest/cypress) - covering your component's functionality.
+
+### Publishing
+By now, you should have all the automatic PR checks passing, 2 manual code reviews approved, and all the UX/accessibility/test QA completed.
+
+NB, the automatic checks are:
+
+* CodeClimate (diff coverage, total coverage, code quality)
+* Travis (runs the tests, [updates the storybook](https://simorghstorybook.now.sh/))
+* Jenkins CI* (`bbc-news-simorgh` job - runs the tests).
+
+\* Currently we run tests on Jenkins with a view to moving away from Travis, so tests are currently run twice. We hope to simplify things soon.
```

Comment:
Soz one small thing again 🙈 Talking to @sareh she wasn't aware moving away from travis is a thing, is it?

Reply:
Discussion on slack with @sareh agreed we should leave this and change when we refactor the jenkins pipeline